### DISCUSSION TITLE
How to find out my own i2p address?

### DISCUSSION THREAD

**Discussion by buhalo1** [markqvist/Reticulum#508] [Help & Questions] [answered]
There is also a problem with connecting to a node whose address is specified in the testnet settings.

g3br23bvx3lq5uddcsjii74xgmn6y5q325ovrkq2zw2wbzbqgbuq.b32.i2p

According to the I2P logs, the daemon reports that the key for this node is invalid.

**Comment by markqvist** [comment]
You can get your own connectable I2P B32 address by running the `rnstatus` command, which will (assuming you have a connectable I2P interface configured) give you output like this:

```
 I2PInterface[I2P]
    Status    : Up
    Peers     : 0 connected I2P endpoints
    Mode      : Gateway
    Rate      : 100.00 kbps
    I2P B32   : g3br23bvx3lq5uddcsjii74xgmn6y5q325ovrkq2zw2wbzbqgbuq.b32.i2p
    Traffic   : 0 B↑
                0 B↓
```

It seems the I2P daemon on that testnet hub had crashed. It should be back up again shortly.

**Comment by buhalo1** [comment]
There are probably a few more problems than it seemed.
Here is the output of rnstatus.

```
    Shared Instance[37428]
    Status    : Up
    Serving   : 0 programs
    Rate      : 1.00 Gbps
    Traffic   : 0 B↑
                0 B↓

 I2PInterfacePeer[I2P to g3br23bvx3lq5uddcsjii74xgmn6y5q325ovrkq2zw2wbzbqgbuq.b3                                                                             2.i2p]
    Status    : Down
    Mode      : Full
    Rate      : 256.00 kbps
    I2P       : Creating Tunnel
    Traffic   : 0 B↑
                0 B↓

 I2PInterface[I2P]
    Status    : Down
    Peers     : 0 connected I2P endpoints
    Mode      : Full
    Rate      : 256.00 kbps
    Traffic   : 0 B↑
                0 B↓

 Transport Instance <dcf656f44f9d20e52f45ae20fd84dffd> running
 Uptime is 0s
```

I'm running this node on a headless server. 
Maybe some settings are needed on the I2P side?

**Reply by buhalo1** [reply, marked as answer]
There were problems with /etc/i2pd/i2pd.conf 
SAM was disabled by default.
After uncommenting and changing the settings, everything started working.
